source/core/output_buffers/ut_output_table_buffer.tpb

```diff
@@ -99,13 +100,26 @@ create or replace type body ut_output_table_buffer is
       commit;
     end;
 
-  begin
-    loop
-      select a.message_id, ut_output_data_row(a.text, a.item_type)
+    procedure remove_buffer_info is
+      pragma autonomous_transaction;
+    begin
+      delete from ut_output_buffer_info_tmp a
+       where a.output_id = self.output_id;
+      commit;
+    end;
+
+    begin
+    while not l_finished loop
+      with ordered_buffer as (
+        select a.message_id, ut_output_data_row(a.text, a.item_type)
+          from ut_output_buffer_tmp a
+         where a.output_id = self.output_id
+         order by a.message_id
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add ASC in order to make the order explicit. ([plsql:OrderByExplicitAscCheck](https://sonarcloud.io/organizations/utplsql/rules#rule_key=plsql%3AOrderByExplicitAscCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=utPLSQL&pullRequest=816&issues=AWhCR0RKSi5GWQS9iTtZ&open=AWhCR0RKSi5GWQS9iTtZ)

[ISSUE_KEY]: AWhCR0RKSi5GWQS9iTtZ